### PR TITLE
translation in French of the tool_for_renewal_en.md

### PR DESCRIPTION
**Pull request by Tomahna81**
This is a translation in French of https://www.monerooutreach.org/stories/the-tool-for-renewal.html

Open for review.

### CODE REVIEW

**Comment by JLB-XMR** [comment]
Following the request of @lh1008, I'm ready for the review.

**Comment by lh1008** [comment]
Thank you @JLB-XMR for taking the review.

**Comment by Tomahna81** [comment]
> Hello @Tomahna81
> 
> Please change the files name to, `tool_for_renewal_fr.md`. That way we keep consistency throughout the repository. The English file's name is, `tool_for_renewal_en..md`, what changes are the language extensions.
> 
> Thank you.

Hi, renaming is done now. Waiting for the comments from the reviewer.

Cheers.

**Comment by lh1008** [comment]
Hey @JLB-XMR 

Like I told you in the chat, you will have to create a new branch for the wallet_primer_fr.md file. Let me know if you need any help with this. We will also delete the unnecessary commits :).

**Comment by Tomahna81** [comment]
> Hey @JLB-XMR
> 
> Like I told you in the chat, you will have to create a new branch for the wallet_primer_fr.md file. Let me know if you need any help with this. We will also delete the unnecessary commits :).

Hi lh1008,
Thanks for the message. I think this was for me (@Tomahna81) and not to @JLB-XMR. In any case, I have removed the wallet_primer_fr.md from this branch. As you advised me, wallet_primer_fr.md is another translation that should go on an independent branch.

**Comment by lh1008** [comment]
Hey @Tomahna81,

Yeah, the past message was for you, oups. I'm currently speaking with the reviewer and helping him start the workflow through GitHub, I believe this week we should have started already so we can continue with the next PR.

**Comment by Hsterofemia** [comment]
[Review_the tool_for_renewal_fr.txt](https://github.com/monero-ecosystem/outreach-docs/files/5507961/Review_the.tool_for_renewal_fr.txt)
Hi Guys, please find attached the recommended comments and changes. The text is great, these are minor changes. Let me know if this is the best format to communicate the changes.

Cheers

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ En 2009 à la suite du grand effondrement financier, un groupe ou une personne s
...
+Cependant, étant tout récent, Bitcoin avait des défauts techniques, incluant une faible confidentialité. Et alors un autre groupe ou personne sous le pseudonyme de Nicolas van Saberhagen développa le protocole CryptoNote. Les développeurs prirent acte et Monero naissait. Monero a résolu plusieurs des problèmes de Bitcoin, incluant le fait qu’il soit facile à surveiller par les entreprises et gouvernements. En tant que société d’analyse des chaînes de blocs, Chainalysis a récemment déclarée dans un webinaire public, [chainalysis.com/.../Advanced-Obfuscation-Techniques-Public-Webinar.mp4](https://go.chainalysis.com/rs/503-FAP-074/images/Advanced-Obfuscation-Techniques-Public-Webinar.mp4), que « Monero est bien plus focalisé sur la confidentialité que ces homologues… Tout dans Monero est masqué, vous ne pouvez distinguer quoique ce soit. »
```
Comment (Hsterofemia) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md]: Chainalysis a récemment déclarée dans un webinaire public ==> Chainalysis a récemment **déclaré** dans un webinaire public. La version originale était la bonne, le PP ne s'accorde pas ici car le COD est placé après

**Review by Hsterofemia — CHANGES_REQUESTED** [review]
Review with corrections, mostly grammar, syntax and conjugation. format is:
[original text] ==> [Modification ; explanation for the suggested modification]

**Comment by Hsterofemia** [comment]
Hi @Tomahna81, I think you maybe missed some comments on lines 17 to 37.

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md (hunk 1 above)
Comment (OthmanB) [reply monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md]: Ok. Pas de soucis.

**Comment by lh1008** [comment]
@Hsterofemia could you please approve the changes so we can merge this pull request? :)

**Comment by lh1008** [comment]
@Tomahna81 once the commits are approved we will have to remove unnecessary commits and squash into one commit the rest. Let me know once this is approved so I can help you out. :)

**Review by Hsterofemia — APPROVED** [review]
Changes approved.

**Review by Hsterofemia — APPROVED** [review]
Changes approved

**Review by Hsterofemia — DISMISSED** [review]

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Que vous ayez besoin de transférer votre argent dans une archive numérique de valeur, privé par nécessité ou que vous souhaitiez sur le principe participer au désengagement du système financier actuel, rejoignez-nous. Ensemble, nous pouvons utiliser la technologie nouvelle pour limiter l’intrusion des banquiers corrompus et des gouvernements, et participer à un système financier donnant le pouvoir au peuple. Nous sommes tous ensemble dans cela. Utilisons ces outils pour être le changement que nous souhaitons voir dans le monde, pour nous même et pour les générations futures.
+
+Nous ne nous attendons pas à ce que vous preniez nos paroles telles quelles et nous vous encourageons à faire votre propre recherche et vérifier les sources. Nous partagerons ici certains points de départ recommandés. Une fois que vous aurez fini de chercher, veuillez revenir ici pour un guidage rapide sur comment commencer à utiliser Monero (voir ci-dessous).
+
+### _Compléments de lecture_ :
+- [Qu’est-ce que Monero ? (Site web officiel)](https://web.getmonero.org/fr/get-started/what-is-monero/)
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md:30]:
```suggestion
- [Qu’est-ce que Monero ? (Site web officiel)](https://www.getmonero.org/fr/get-started/what-is-monero/)
```

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- [Pourquoi Monero plutôt que Bitcoin (Monero.how)](monero.how/why-monero-vs-bitcoin)
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md:31]:
```suggestion
- [Pourquoi Monero plutôt que Bitcoin (Monero.how)](https://monero.how/why-monero-vs-bitcoin)
```

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- [Pourquoi Monero ? (Monero World)](moneroworld.com/whymonero.html)
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md:32]:
```suggestion
- [Pourquoi Monero ? (Monero World)](https://moneroworld.com/whymonero.html)
```

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+- [Monero en bref](https://www.monerooutreach.org/quick-facts.html)
+- [Monero signifie Argent : Cryptocurrency 101, en direct de Leipzig (Documentaire)](communityworkgroup.org/monero-means-money)
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md:34]:
```suggestion
- [Monero signifie Argent : Cryptocurrency 101, en direct de Leipzig (Documentaire)](https://communityworkgroup.org/monero-means-money)
```

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+Nous ne nous attendons pas à ce que vous preniez nos paroles telles quelles et nous vous encourageons à faire votre propre recherche et vérifier les sources. Nous partagerons ici certains points de départ recommandés. Une fois que vous aurez fini de chercher, veuillez revenir ici pour un guidage rapide sur comment commencer à utiliser Monero (voir ci-dessous).
+
+### _Compléments de lecture_ :
+- [Qu’est-ce que Monero ? (Site web officiel)](https://web.getmonero.org/fr/get-started/what-is-monero/)
+- [Pourquoi Monero plutôt que Bitcoin (Monero.how)](monero.how/why-monero-vs-bitcoin)
+- [Pourquoi Monero ? (Monero World)](moneroworld.com/whymonero.html)
+- [Monero en bref](https://www.monerooutreach.org/quick-facts.html)
+- [Monero signifie Argent : Cryptocurrency 101, en direct de Leipzig (Documentaire)](communityworkgroup.org/monero-means-money)	
+- [“Mastering Monero” (Livre)](masteringmonero.com)
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md:35]:
```suggestion
- [“Mastering Monero” (Livre)](https://masteringmonero.com)
```

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+- [“Zero to Monero” (Livre)](getmonero.org/fr/library/Zero-to-Monero-2-0-0.pdf)
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md:36]:
```suggestion
- [“Zero to Monero” (Livre)](https://www.getmonero.org/fr/library/Zero-to-Monero-2-0-0.pdf)
```

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+**Plateforme d’échange.** Si vous avez un compte bancaire ou avez déjà d’autres crypto-monnaies, vous pouvez utiliser des plateformes d’échanges réputées tel que [Binance.com](https://www.binance.com/) ou [Kraken.com](https://www.kraken.com/) pour acquérir Monero. Le symbole de Monero pour échanger est XMR. Avec une banque, vous pouvez transférer des fonds directement à partir de votre banque vers la plateforme d’échange, acheter des XMR, et transférer vos XMR dans votre propre portefeuille Monero. Veuillez noter que certaines plateformes d’échanges n’ont que XMR en pair avec une autre crypto-monnaie, de telle sorte que vous ayez peut-être besoin de d’abord acheter une autre crypto-monnaie et ensuite la convertir en XMR. Trouvez des plateformes de changes qui listent Monero :
+- [https://web.getmonero.org/fr/community/merchants/index.html](https://web.getmonero.org/fr/community/merchants/index.html)
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md:42]:
```suggestion
- [https://web.getmonero.org/fr/community/merchants/index.html](https://www.getmonero.org/fr/community/merchants/index.html)
```

File: monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+**Vendeurs privés.** Vous pouvez acheter et vendre du Monero auprès de vendeurs privés, en ligne ou en personne, en utilisant [LocalMonero.co](https://localmonero.co/)
+
+**Distributeur automatique de Crypto.** Vous pouvez acheter des XMR sans passer par une banque en utilisant une carte de débit ou crédit dans des distributeurs automatiques de crypto-monnaies tout autour du globe. Ceux-ci proposent souvent des frais plus élevés, mais sont pratiques. Pour en localiser un près de vous, utilisez un service tel que [CoinATMRadar.com](https://coinatmradar.com/) et filtrer la carte par monnaie : Monero (XMR) et Operations : Acheter.
+
+**Accepter Monero (pour entreprises).** La communauté globale de développeurs derrière Monero a créé des outils pour permettre aux entreprises d’accepter des XMR pour des produits et services. Supprimez de l’équation le couteux intermédiaire et laissez vos clients vous payer quelques centimes de frais de transaction. Apprenez en plus : 
+- [Services Marchand de "Monero Outreach"](https://www.monerooutreach.org/merchants/)
+- https://web.getmonero.org/community/merchants/
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/Translations-fr-tool_for_renewal.md:52]:
```suggestion
- https://www.getmonero.org/community/merchants/
```

**Review by erciccione — CHANGES_REQUESTED** [review]

**Comment by erciccione** [comment]
> Please change the files name to, tool_for_renewal_fr.md

This still needs to be changed.

**Review by lh1008 — DISMISSED** [review]
This looks good. :)

**Review by erciccione — CHANGES_REQUESTED** [review]
> Please change the files name to, tool_for_renewal_fr.md

**Review by erciccione — APPROVED** [review]
Looks ok.

**Pull request merged**